identity/backend/src/main/java/io/camunda/identity/usermanagement/service/MembershipService.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.usermanagement.service;
+
+import io.camunda.identity.usermanagement.CamundaUser;
+import io.camunda.identity.usermanagement.Group;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MembershipService {
```

Comment:
I did the separation because I preferred to have a central place to look for all membership related stuff and avoid adding dependency to groups in UserService and vice versa. For example "addUserToGroup" can be in both UserService or GroupService but with this specific MembershipService it's easier to find.

Reply:
Thats fair :) - I agree, some methods can easily be in multiple places so moving it out like this creates a nice intentional place for it